src/pages/Appointments/components/MultiPractitionerSelect.tsx

```diff
@@ -66,7 +66,7 @@ export const MultiPractitionerSelector = ({
   );
 
   const getItemValue = (user: UserBase) => {
-    return formatName(user) + " " + user.id;
+    return formatName(user) + " " + user.username;
```

Comment:
```suggestion
    return `${formatName(user)} ${user.username}`;
```

pointing out the same again 😅